specification/billing/resource-manager/Microsoft.Billing/stable/2023-04-01/examples/billingSavingsPlansList.json

```diff
@@ -0,0 +1,255 @@
+{
+  "parameters": {
+    "billingAccountName": "00000000-0000-0000-0000-000000000000:00000000-0000-0000-0000-000000000000_2019-05-31",
+    "api-version": "2023-04-01",
+    "$filter": "billingProfileId eq '/providers/Microsoft.Billing/billingAccounts/00000000-0000-0000-0000-000000000000:00000000-0000-0000-0000-000000000000_2019-05-31/billingProfiles/AAAA-BBBB-CCC-DDD'",
+    "$orderby": "properties/displayName asc",
+    "refreshSummary": "true",
+    "selectedState": "Succeeded",
+    "take": "3"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "value": [
+          {
+            "id": "/providers/Microsoft.Billing/billingAccounts/00000000-0000-0000-0000-000000000000:00000000-0000-0000-0000-000000000000_2019-05-31/savingsPlanOrders/20000000-0000-0000-0000-000000000000/savingsPlans/30000000-0000-0000-0000-000000000000",
+            "name": "30000000-0000-0000-0000-000000000000",
+            "type": "microsoft.billing/billingAccounts/savingsPlanOrders/savingsPlans",
+            "sku": {
+              "name": "Compute_Savings_Plan"
+            },
+            "properties": {
+              "billingScopeId": "/subscriptions/10000000-0000-0000-0000-000000000000",
```

Comment:
Wouldn't this be a subscription always? if so, should we rename this to billingSubscriptionId? 
Also i believe this is not the right ARM ID for billingSubscription

Reply:
billing scope id is not always subscription id for savings plan, it could also be something like "/providers/Microsoft.Billing/billingAccounts/1234567/billingSubscriptions/30000000-0000-0000-0000-000000000003". One is for RBAC scenario and the other one is for billing admin scenario. When user makes the call to get it in billing admin scenario(through Microsoft.Billing) we will return the value however the savings plan was purchased.

See https://learn.microsoft.com/en-us/azure/cost-management-billing/savings-plan/buy-savings-plan#purchase-with-the-savings-plan-order-alias---create-api